pkgs/applications/office/gnucash/default.nix

```diff
@@ -26,12 +27,12 @@
 
 stdenv.mkDerivation rec {
   pname = "gnucash";
-  version = "5.3";
+  version = "5.4";
 
   # raw source code doesn't work out of box; fetchFromGitHub not usable
   src = fetchurl {
     url = "https://github.com/Gnucash/gnucash/releases/download/${version}/${pname}-${version}.tar.bz2";
```

Comment:
404 🤔

Reply:
Huh, they reuploaded a separate `-1` tarball, but on the same release.
I'll leave the version on 5.4 as that's still the upstream release, but add a -1 on the fetchurl for now.

Reply:
https://lists.gnucash.org/pipermail/gnucash-announce/2023-September/000398.html
apparently it was this, they had an issue in their release process.